python/lsst/pipe/base/graph/_versionDeserializers.py

```diff
@@ -474,14 +489,25 @@ def readHeaderInfo(self, rawHeader: bytes) -> SimpleNamespace:
         infoMappings.dimensionRecords = {}
         for k, v in infoMap["DimensionRecords"].items():
             infoMappings.dimensionRecords[int(k)] = SerializedDimensionRecord(**v)
+        # This is important to be a get call here, so that it supports versions
+        # of saved quantum graph that might not have a saved universe without
+        # changing save format
+        if (universeConfig := infoMap.get("universe")) is not None:
+            universe = DimensionUniverse(config=DimensionConfig(universeConfig))
+        else:
+            universe = DimensionUniverse()
```

Comment:
I worry a bit that we should be using a universe that was passed into the loader if such a universe exists, although I imagine this code path is never going to trigger because historically people haven't been persisting their graph so that they can load it again 6 months later.